.github/workflows/publish.yml

```diff
@@ -59,20 +57,30 @@ jobs:
         with:
           node-version: '16.13.0'
           registry-url: "https://npm.pkg.github.com"
-          # Defaults to the user or organization that owns the workflow file
           scope: "@zondax"
       - name: Install yarn
         run: npm install -g yarn
-      - name: Install modules
-        run: yarn install
       - name: Build package
-        run: yarn build
+        run: |
+          yarn install
+          yarn build
       - name: Get latest release version number
         id: get_version
         uses: battila7/get-version-action@v2
-      - name: Update tag
+      - name: Show version
+        run: echo ${{ steps.get_version.outputs.version }}
+      - name: Clean latest release version number
+        id: get_version_cleaned
+        uses: bhowell2/github-substring-action@v1
+        with:
+          output_name: version
+          value: ${{ steps.get_version.outputs.version }}
+          index_of_str: "npm_v"
```

Comment:
```suggestion
```
we should delete this whole section that removes `npm_v` this is not useful in this context